src/backend/utilities/batchSignAndSubmitAttestation.ts

```diff
@@ -85,18 +88,22 @@ async function createPendingTransaction() {
   }
   logger.debug('Scheduling next transaction');
 
-  const extrinsics = currentAttestations.map(
+  const newAttestations = currentAttestations.map(
     ({ attestation: { cTypeHash, claimHash } }) =>
       api.tx.attestation.add(claimHash, cTypeHash, null),
-  );
+  ) as SubmittableExtrinsic[];
+
+  // TODO: manage which blocks to choose
+  const submittableRevocations = blackList.slice(0, 100);
```

Comment:
The magic number `100` should rather be a named constant. Also, I suggested that during development we reduce this number to 1 or 2, to avoid occasionally expiring everything and having no items to work with.

Reply:
okk

Reply:
done